nni/experiment/config/local.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from dataclasses import dataclass
+from typing import List, Optional, Union
+
+from .common import TrainingServiceConfig
+
+__all__ = ['LocalConfig']
+
+@dataclass(init=False)
+class LocalConfig(TrainingServiceConfig):
```

Comment:
we also need this kind of class for other training services, right?

Reply:
Yes. In experiment config PR, not this one.